docs/USAGE.md

```diff
@@ -0,0 +1,282 @@
+# Usage
+
+**WARNING**: Please notice that no official release has been created for this repo yet, so installation instructions below would require you to build the code manually beforehand.
+
+## Configure the OpenTelemetry Tracing Library for .NET
+
+### Configuration values
+
+Use these environment variables to configure the tracing library:
+
+| Environment variable | Default value | Description |
+|-|-|-|
+| `OTEL_SERVICE_NAME` |  | The name of the service. |
```

Comment:
Should it not be `OTEL_SERVICE` ?

Reply:
I tend to prefer `OTEL_SERVICE_NAME` - eventually this should come from resources, anyway, I think that the suffix `_NAME` makes it a bit more intuitive. Happy to be convinced otherwise.

Reply:
I don't what to change that in this PR as this is purely documentation issue. Please create GH issue if you feel the other name would be more fitting.